ci-operator/step-registry/baremetal/lab/ipi/conf/external-lb/baremetal-lab-ipi-conf-external-lb-commands.sh

```diff
@@ -25,4 +25,7 @@ platform:
       type: UserManaged
 EOF
 
+# Backup vips.yaml file to move it back after the install-config.yaml is
+# processed in the install step
+cp "$SHARED_DIR/vips.yaml" "$SHARED_DIR/vips_backup_for_ipi_external_lb.yaml"
```

Comment:
Can we rather limit the changes for this bugfix to the provisioning network pre and post steps?

Reply:
This file is the pre step in which the vips.yaml file is overwritten by the external_vips.yaml file. I had added the reverting of the file in the ipi-install step to not lose it in case the cluster is preserved, prune_nodes would use the overwritten file and fail to delete the correct vlan interface. I have updated to save the backup on bastion now so we should have it in case the cluster is preserved. Vlan is already created with the last octet of the first master node. It is just overwritted with the external_vips.yaml for external_lb install.

Reply:
Yes, what I'm saying is that if we can get that Id from the hosts.yaml instead of the API_VIP, we may decouple from the vips.yaml file that can change in several cases or be actually uneffective (that external lb one, the disconnected ipi case, ipv6, some hypershift scenarios).

How much would be the effort of changing this reference in the pre step of the provisioning network and in the post one? Do we also need to change it in the pruner script?

Reply:
Done